docs/dev-concepts/faq.md

```diff
@@ -87,21 +88,25 @@ There are no message storage and retrieval-related fees incurred by developers f
 
 ### How does XMTP establish a secure and fraud-proof relationship between two identities?
 
-Blockchain accounts sign and advertise a set of keys that XMTP uses to establish a shared secret for encryption using another blockchain account’s keys. These keys attest to the authenticity of both accounts and are required to add messages to their conversation. No third-party apps or relayers are involved in this process.
+Blockchain accounts sign and advertise a set of keys to start using XMTP. These keys attest to the authenticity of both accounts. XMTP uses these keys to establish a shared secret between the blockchain accounts and uses it to generate a key used to encrypt an invitation that allows the blockchain accounts to start exchanging messages. No third-party apps or relayers are involved in this process.
```

Comment:
@mkobetic I made these refinements to reflect XMTP V2 - any thoughts?

Reply:
I would drop the `These keys attest to the authenticity of both accounts.`. It's not clear what `both accounts` means and doesn't seem necessary. I would also split the long sentence in the middle, it's a bit long to follow.